client/src/components/Focusedzone/Focusedzone.js

```diff
@@ -64,9 +64,9 @@ class Focusedzone extends Component {
     this.wasClicked = true;
   }
 
-  handleDocumentClick() {
+  handleDocumentClick(evt) {
```

Comment:
```suggestion
  handleDocumentClick(event) {
```
There's no need to shorten this, `event` is already short and is more immediately clear what it is.

Reply:
Technically you can use `event`, though I don't like it because of confusion of whether it clashes with `window.Event`. Easier to avoid the confusion

Reply:
Discussed offline - while `event` is _way_ more common in our codebase, a few `evt` have already snuck in so we may as well allow this here too.